docs/tutorials/podman_tutorial.md

```diff
@@ -46,8 +46,8 @@ podman inspect -l | grep IPAddress\":
             "IPAddress": "",
 ```
 
-Note: The -l is a convenience argument for **latest container**.  You can also use the container's ID instead
-of -l.
+Note: The -l is a convenience argument for **latest container**. This option is not available with the remote Podman client, so you have to instead
```

Comment:
There's a War Against The Y-Word, and if the main activists notice this they'll block it. Suggested wording:
>This option is not available with the remote Podman client; use the container name or ID instead.

Reply:
I like your suggestion and applied it. Could someone please squash the commits?